build.sbt

```diff
@@ -17,7 +17,7 @@ libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-core" % sparkVersion % "provided",
   "org.apache.spark" %% "spark-sql" % sparkVersion % "provided",
   "com.github.music-of-the-ainur" %% "almaren-framework" % s"0.9.8-${majorVersion}" % "provided",
-  "org.elasticsearch" %% "elasticsearch-spark-20" % "8.4.3" % "provided",
+  "org.elasticsearch" %% "elasticsearch-spark-20" % "8.4.2" % "provided",
```

Comment:
We can have 8.4.3 version with no problem, only the connector version should be 0.0.2